pkg/river/token/builder/builder.go

```diff
@@ -131,6 +132,10 @@ func (b *Body) AppendBlock(block *Block) {
 // Body. If any value reachable from goValue implements Tokenizer, the printed
 // tokens will instead be retrieved by calling the RiverTokenize method.
 //
+// Optional attributes and blocks may be trimmed. If the goValue implements
+// Defaulter, any values matching the defaults will be trimmed. If not, any
+// values matching the golang defaults will be trimmed.
```

Comment:
WDYT about this rewrite to make the language stronger about trimming?

```suggestion
// Optional attributes and blocks set to default values are trimmed. 
// If goValue implements Defaulter, default values are retrieved by 
// calling SetToDefault against a copy. Otherwise, default values are 
// the zero value of the respective Go types.
```